metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py

```diff
@@ -129,11 +129,6 @@ class PrestoOnHiveConfig(BasicSQLAlchemyConfig):
         description="Add the Presto catalog name (e.g. hive) to the generated dataset urns. `urn:li:dataset:(urn:li:dataPlatform:hive,hive.user.logging_events,PROD)` versus `urn:li:dataset:(urn:li:dataPlatform:hive,user.logging_events,PROD)`",
     )
 
-    extra_properties: List[str] = Field(
```

Comment:
I removed this at all as nobody uses this property, and it didn't even work in the past.